psi4/src/psi4/libmints/oeprop.cc

```diff
@@ -2381,6 +2421,24 @@ PopulationAnalysisCalc::compute_mbis_multipoles(bool print_output) {
 
     }
 
+    const int power = options.get_int("RADIAL_MOMENT");
+    
+    auto avols = compute_atomic_volumes(grid, rho_a, distances, num_atoms);
+    
+    std::stringstream sstream;
+
+    sstream << "MBIS RADIAL MOMENTS <R^" << power << ">";
+
+    auto var_name = sstream.str();
+    
+    wfn_->set_array_variable(var_name, avols);
+
+    auto valence_widths = std::make_shared<Matrix>("MBIS Valence Widths", num_atoms, 1);
+    for (int atom = 0; atom < num_atoms; atom++) {
+        valence_widths->set(atom, 0, Sai[atom][mA[atom]-1]);
```

Comment:
It may be more convenient to make the first dimension the 1 (https://github.com/psi4/psi4/blob/master/psi4/src/psi4/libmints/oeprop.cc#L1749). then if these are per-atom quantities, can add to list https://github.com/psi4/psi4/blob/master/psi4/driver/p4util/python_helpers.py#L666 so the qcvar comes to user as 1D array. ping me if I'm not being clear.

Reply:
Well, the MBIS charges are given in the num_atoms x 1 format. I think it's better if we keep everything consistent